src/collections/lookup-map.ts

```diff
@@ -1,23 +1,42 @@
 import * as near from '../api'
 import { Bytes } from '../utils';
+import { Serializer } from 'superserial';
 
 export class LookupMap {
     readonly keyPrefix: Bytes;
+    readonly serializer: Serializer;
 
-    constructor(keyPrefix: Bytes) {
+    constructor(keyPrefix: Bytes, classes?: {
+        [className: string]: ((new (...args: any[]) => any) | Function);
+    }) {
         this.keyPrefix = keyPrefix
+        this.serializer = new Serializer({classes})
     }
 
     containsKey(key: Bytes): boolean {
         let storageKey = this.keyPrefix + key
         return near.storageHasKey(storageKey)
     }
 
+    containsObjectKey(key: any): boolean {
+        let storageKey = this.keyPrefix + this.serializer.serialize(key)
+        return near.storageHasKey(storageKey)
+    }
+
     get(key: Bytes): Bytes | null {
         let storageKey = this.keyPrefix + key
         return near.storageRead(storageKey)
     }
 
+    getObject(key: any): any {
+        let storageKey = this.keyPrefix + this.serializer.serialize(key)
+        let raw = near.storageRead(storageKey)
+        if (raw !== null) {
+            return this.serializer.deserialize(raw)
+        }
+        return null
+    }
```

Comment:
I don't personally see the reason for having both supported like this. The whole use case for JS SDK is that it is easier to use. I have a strong but low confidence opinion that the lookup map should be typed as `LookupMap<K, V>` where the keys and values would be serialized and have that as the base methods (`get`, `remove`, etc.). If the types are generic, you shouldn't need this superserial library, do you?

If the duplication is kept, I don't see why this wouldn't be swapped where the more primitive ones would be called something else, and the simple methods to use are the name like `get`